src/config/polyfills.js

```diff
@@ -0,0 +1,3 @@
+global.Headers = require('fetch-headers');
```

Comment:
It's a little odd to me to be assigning this to `global` and just using the fact that our webpack setup has aliased `global` to `window` to be compatible with node